main.py

```diff
@@ -3,14 +3,27 @@
 # throughout this file
 import pygame
 
+from constants import SCREEN_HEIGHT, SCREEN_WIDTH
+
 
 def main():
-    try:
-        pygame.init()
-        print('Starting asteroids!')
-        pygame.quit()
-    except Exception as e:
-        print(f'Failed to initialize Pygame: {e}')
+    pygame.init()  # Initialize Pygame
+
+    # Set up the display
+    screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+    
+    # Start the game loop
+    while True:
+        # Event handling to check if the user wants to quit
+        for event in pygame.event.get():
+            if event.type == pygame.QUIT:
+                return  # Exit the game loop if the user closes the window
+
+        # Fill the screen with black color (RGB: 0, 0, 0)
+        screen.fill((0, 0, 0))
+
+        # Update the display
```

Comment:
**Suggestion:** Add a delay to control the game loop's execution speed.

**Existing Code:**
```Python
# Update the display
```

**Improved Code:**
```Python
pygame.display.flip()
pygame.time.delay(10)  # Add a small delay to reduce CPU usage
```

**Details:**
Consider adding a delay or a sleep mechanism to control the game loop's execution speed, which can help in managing CPU usage.